apps/andi/mix.exs

```diff
@@ -54,7 +54,7 @@ defmodule Andi.MixProject do
       {:placebo, "~> 1.2", only: [:dev, :test, :integration]},
       {:plug_cowboy, "~> 2.1"},
       {:simply_validate, ">= 0.2.0"},
-      {:smart_city, "~> 3.0", override: true},
+      {:smart_city, "~> 3.11"},
```

Comment:
does this need to be specifically 3.11?  pinning to 3.0 should suffice here.

Reply:
We were removing the override true in places it didn't need to be and just put the current version. We'll switch this to 3.0.